RunLog/RunLog/Sources/Data/Network/API.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  API.swift
+//  RunLog
+//
+//  Created by 심근웅 on 3/19/25.
+//
+
+import Foundation
+
+//final class API {
+//    static let openWeatherURL: String = "https://api.openweathermap.org/data/2.5"
+//}
+enum API: String {
+    case openWeatherURL = "https://api.openweathermap.org/data/2.5"
```

Comment:
```suggestion
   static let openWeatherURL = "https://api.openweathermap.org/data/2.5"
```
enum 내에서도 static let을 사용할 수 있습니다.
이렇게하면 .rawValue 를 안하고도 가져올 수 있기 때문에 이 방식이 더 좋아보입니다.

Reply:
이런 방법이 있었네요 수정했습니다!